core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -42,6 +44,10 @@ oppia.directive('questionsTab', [
             $scope.canEditQuestion = $scope.topicRights.canEditTopic();
             $scope.questionSummaries =
               TopicEditorStateService.getQuestionSummaries();
+            $scope.questionSuggestionThreads = [];
+            $scope.getSuggestedQuestions();
+            $scope.questionUnderReview = null;
```

Comment:
$scope.activeQuestion or $scope.questionCurrentlyBeingReviewed perhaps. In some sense all these questions are currently "under review".

Reply:
Done